benchmarks/stb_stbi_read_fuzzer/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -eu
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# remove this line when this change is upstreamed
+sed '2d' $SRC/stb/tests/stb_png_read_fuzzer.cpp > $SRC/stb/tests/stbi_read_fuzzer.c
+
+$CXX $CXXFLAGS -std=c++11 -I. -DSTBI_ONLY_PNG  \
```

Comment:
This fuzzer is not needed, can you remove this in another PR.

Reply:
Do you mean the line? Otherwise I am not sure what fuzzer you're referring to

Reply:
yes this line only, since stb_png_read_fuzzer is not used anywhere, also remove its dict use at line 32.

$CXX $CXXFLAGS -std=c++11 -I. -DSTBI_ONLY_PNG  \
     $SRC/stb/tests/stbi_read_fuzzer.c \
    -o $OUT/stb_png_read_fuzzer $LIB_FUZZING_ENGINE